submissions/vovan-zt/app_friends/script.js

```diff
@@ -0,0 +1,171 @@
+
+window.addEventListener('DOMContentLoaded', () => {
+
+    async  function getResponse () {
+        let response = await fetch('https://randomuser.me/api/?results=20');
+        let content = await response.json(); 
+
+        const {results} = content;
+        let dataBase = [...results];
+
+        function render (data) {
+            data.forEach (item => {
+                let card = document.createElement('div')
+                card.classList.add('cards__wrapper');
+              
+                card.innerHTML = `
+                    <div class="cards__wrapper-name">
+                        <h3> ${item.name.first} ${item.name.last}</h3>
+                    </div>
+
+                    <div class="cards__wrapper-photo">
+                        <img src=${item.picture.large} alt="photo">
+                    </div>
+
+                    <div class="cards__wrapper-years">
+                    I have ${item.dob.age} years old
+                    </div>
+
+                    <div class="cards__wrapper-email">
+                        ${item.email}
+                    </div>
+
+                    <div class="cards__wrapper-phone">
+                        ${item.cell}
+                    </div>
+
+                    <div class="cards__wrapper-city">
+                        ${item.location.city}
+                    </div>
+
+                    <hr class="devider">
+
+                    <div class="cards__wrapper-sex">${item.gender}</div>
+                
+                `
+                document.querySelector('.cards').appendChild(card);
+               
+            });
+
+               
+            document.querySelectorAll('.cards__wrapper-sex').forEach(item => {
+                if (item.innerHTML =="female") {
+                    item.classList.add('color_violet');
+                    item.parentNode.firstElementChild.classList.add('bgcolor_violet'); 
+                } 
+            });
+        }
+        render (dataBase);
+
+        let cardsWrapper = document.querySelectorAll('.cards'),
+            ageUp = document.querySelector('.filter__age-up'),
+            ageDown = document.querySelector('.filter__age-down'),
+            ageSearch = document.querySelector('.filter__age-searchTerm'),
+            nameUp = document.querySelector('.filter__name-up'),
+            nameDown = document.querySelector('.filter__name-down'),
+            nameSearch = document.querySelector('.filter__name-searchTerm'),
+            male = document.querySelector('#male'),
+            female = document.querySelector('#female'),
+            allPeople = document.querySelector('#all'),
+            reset = document.querySelector('.filter__reset-button');
+
+        function deleteDataBase () {
```

Comment:
On line 9 you have an array of user cards with the name `dataBase` reading this function your colleagues will think that this function deletes all data in this array. Try to add a more appropriate function name. In case you want to clear all cards from the page just clear the content of the cards wrapper in one line instead of looping through all cards.